### PR TITLE
Benchmarks: Build Pipeline - Call build script when setup environment.

### PR DESCRIPTION
**Pull request by guoshzhao**
**Description**
Call build script when setup environment.

**Major Revision**
- Call build script of benchmarks in Makefile, the command is `make cppbuild`
- Add cppbuild command for testing env.
- Add cppbuild command for docker image.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,3 +10,6 @@ cpplint:
 
 cppformat:
 	clang-format --verbose -i $(CPPSOURCES)
+
+cppbuild:
+	cd ./superbench/benchmarks/; bash build.sh
```
Comment (abuccts) [inline Makefile:15]: change `;` to `&&`? why not `./superbench/benchmarks/build.sh` directly?

Comment (guoshzhao) [reply Makefile:15]: Both makefile and build.sh use relative path. If use `./superbench/benchmarks/build.sh`, the script will loop `./superbenchmark/` dir not `./superbenchmark/superbench/benchmarks`

**Review by cp5555 — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -10,3 +10,6 @@ cpplint:
 
 cppformat:
 	clang-format --verbose -i $(CPPSOURCES)
+
+cppbuild:
+	cd ./superbench/benchmarks/ &&  bash build.sh
```
Comment (abuccts) [inline Makefile:15]: two spaces after `&&`

File: .azure-pipelines/cuda-unit-test.yml [hunk 3]
```diff
@@ -15,6 +15,9 @@ steps:
   - script: |
       python3 -m pip install .[test,torch]
     displayName: Install dependencies
+  - script: |
+      make cppbuild
+    displayName: Build benchmarks
```
Comment (abuccts) [inline .azure-pipelines/cuda-unit-test.yml:20]: should we build these in unit test? there's no unit test for cpp code, and build process will be checked in docker build ci.

May be build and run these after there's integration tests?

**Review by abuccts — APPROVED** [review]

File: .azure-pipelines/cuda-unit-test.yml (hunk 3 above)
Comment (guoshzhao) [reply .azure-pipelines/cuda-unit-test.yml:20]: The cppbuild inside cuda-unit-test.yml is for python benchmarks, we should check something like 'if binary exists' and 'the output format'.

File: Makefile (hunk 2 above)
Comment (guoshzhao) [reply Makefile:15]: Thanks

**Pull request merged**